### ISSUE TITLE
Not compiling on Elixir 1.7-dev

### ISSUE THREAD

**Issue by mindreframer** [obmarg/kazan#18]
```
==> kazan
Compiling 16 files (.ex)
warning: variable "other" is unused
  lib/kazan/server.ex:186


== Compilation error in file lib/kazan/models.ex ==
** (UndefinedFunctionError) function String.Casing.titlecase_once/1 is undefined or private. Did you mean one of:

      * titlecase_once/2

    (elixir) String.Casing.titlecase_once("apis")
    lib/kazan/codegen/models.ex:58: anonymous fn/1 in Kazan.Codegen.Models.module_name/2
    (elixir) lib/enum.ex:1296: Enum."-map/2-lists^map/1-0-"/2
    lib/kazan/codegen/models.ex:55: Kazan.Codegen.Models.module_name/2
    (elixir) lib/enum.ex:745: anonymous fn/3 in Enum.each/2
    (stdlib) lists.erl:1263: :lists.foldl/3
    (elixir) lib/enum.ex:1917: Enum.each/2
    lib/kazan/codegen/models.ex:101: Kazan.Codegen.Models.parse_models/1
could not compile dependency :kazan, "mix compile" failed. You can recompile this dependency with "mix deps.compile kazan", update it with "mix deps.update kazan" or clean it with "mix deps.clean kazan"
```

**Comment by mindreframer** [comment]
Output from  `iex`: 

```
iex(1)> String.Casing.titlecase_once("apis")
** (UndefinedFunctionError) function String.Casing.titlecase_once/1 is undefined or private. Did you mean one of:

      * titlecase_once/2

    (elixir) String.Casing.titlecase_once("apis")
iex(1)> String.Casing.titlecase_once("apis", 0)
{"A", "pis"}
```

**Comment by obmarg** [comment]
Hey @mindreframer - thanks for the bug report.  This is already fixed on `master` - I'll be putting together a release soon, but in the meantime if you want to pull direct from github anything after  `a960c51747` should work fine.

**Comment by obmarg** [comment]
Actually, I'm just going to release a patch version w/ the elixir 1.6.0+ fix just now.  Was going to wait till I'd had a chance to manually test the kubernetes 1.8 changes, but I suppose that can wait for a future release.

**Comment by obmarg** [comment]
Fix for this has been released in v0.5.1

**Comment by mindreframer** [comment]
Great, thanks a lot!
